### ISSUE TITLE
How do I make comment questions mandatory

### ISSUE THREAD

**Issue by djmoffat** [BrechtDeMan/WebAudioEvaluationTool#218]
Eg. I have a comment radio with two options, how do I make one comment question manditory?

**Comment by nickjillings** [comment]
Each survey element has the "mandatory" tag:
```xml
<surveyelement type="radio" id="unique_id" mandatory="true">
...
</surveyelement>
```

**Comment by nickjillings** [comment]
Implemented in 8fa6d39. The `commentquestion`, `commentradio`, `commentquestion` entries all have a mandatory tag. The `commentslider` also has a mandatory but cannot be checked since it always returns a value!

```xml
<commentcheckbox id="test-1" mame="question-1">
    <statement>Test</statement>
    <option name="A">A</option>
    <option name="B">B</option>
</commentcheckbox>
```
